LayoutTests/imported/w3c/web-platform-tests/content-security-policy/reporting/report-clips-sample.https-expected.txt

```diff
@@ -1,7 +1,5 @@
 
 PASS Unsafe eval violation sample is clipped to 40 characters.
-FAIL Function constructor - the other kind of eval - is clipped. assert_throws_js: function "_ => {
-      new Function("a", "b", "return '1234567890123456789012345678901234567890';");
-    }" did not throw
+FAIL Function constructor - the other kind of eval - is clipped. assert_equals: expected "Function|(a,b) {return '12345678901234567890123" but got "Function|function anonymous(a,b) {return '12345"
```

Comment:
This behaviour needs discussing to come to a consensus on correct behaviour. https://github.com/w3c/trusted-types/issues/491